dspace-api/src/main/resources/spring/spring-dspace-addon-import-services.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans.xsd
+           http://www.springframework.org/schema/context
+           http://www.springframework.org/schema/context/spring-context.xsd"
+       default-autowire-candidates="*Service,*DAO,javax.sql.DataSource">
+
+    <context:annotation-config /> <!-- allows us to use spring annotations in beans -->
+
+    <bean id="importService" class="org.dspace.importer.external.service.ImportService" scope="singleton"
+          lazy-init="false" autowire="byType" destroy-method="destroy"/>
+
+    <bean id="ScidirImportService" class="org.dspace.importer.external.scidir.ScidirImportSourceServiceImpl" scope="singleton">
+        <property name="metadataFieldMapping" ref="ScidirMetadataFieldMapping"/>
+        <property name="apiUrl" value="http://api.elsevier.com/content/search/scidir"/>
```

Comment:
TODO (note to myself): Once #1104 is merged, this "apiUrl" setting could simply pull in its value from the configuration file, instead of hardcoding it again in this location.  The #1104 feature allows us to reference configuration settings from Spring XML configs